src/participant.rs

```diff
@@ -25,16 +25,87 @@ struct ProgressIndex {
     sid: Identifier,
 }
 
+/// Possible outcomes from processing one or more messages.
```

Comment:
I think a little more documentation here would be useful in _how_ `ProcessOutcome` will be used. From the documentation here it's not entirely clear it's purpose (the documentation for `process_message` helps but I think it'd still be valuable to document how this type is used here).

Reply:
Hm, I added some additional documentation about what the various types mean, but didn't really get at what you're supposed to do with it if you get it back from a protocol execution. I could add a note in #180 or elsewhere to update it once we actually do something with protocol outputs, but right now we just ignore them.

Is there another question I could try to answer here?

Reply:
That makes sense. Then maybe just adding a "TODO: Documentation to be fleshed out in #180" would be good.